### PR TITLE
minimal documentation

### PR DESCRIPTION
**Pull request by christophsax**
also included x13binary-package.Rd and NAMESPACE to roxygen

### CODE REVIEW

**Comment by eddelbuettel** [comment]
Uggg. I have such a distrust of complex toolchains (whose authors may have a well-earned reputation of making non-backwards compatible changes) that I never _ever_ let a tool write `NAMESPACE` or `DESCRIPTION`.   

Well at least not in the packages I write by myself :)  And this one is tiny.  So thanks for taking care of it.

File: NAMESPACE [hunk 1]
```diff
@@ -1 +1,5 @@
-exportPattern("^[[:alpha:]]+")
+# Generated by roxygen2 (4.1.1): do not edit by hand
```
Comment (eddelbuettel) [inline NAMESPACE:1]: Worth upgrading.  My machines all have roxygen 5.0.1.

**Comment by christophsax** [comment]
we won't let roxygen working on `DESCRIPTION`. And our `NAMESPACE` seems to be simple enough.

**Comment by eddelbuettel** [comment]
:+1:

roxygen2 5.0.\* still has (had?) a bug that on the first run after the upgrade it rewrites your DESCIPTION (ugghhh).  After that it just uses it to mark its version there rather than in each .Rd file. And agreed on this NAMESPACE.

**Pull request merged**